tendrl/commons/jobs/__init__.py

```diff
@@ -126,14 +126,21 @@ def _run(self):
                             Event(
                                 ExceptionMessage(
                                     priority="error",
-                                    job_id=job.job_id,
```

Comment:
if we use it is stored as a job_update then it never present in event table, because event and job updates are different.